cbmc/proofs/IotMqtt_UnsubscribeAsync/IotMqtt_UnsubscribeAsync_harness.c

```diff
@@ -0,0 +1,102 @@
+#include "iot_config.h"
+#include "private/iot_mqtt_internal.h"
+
+#include <stdlib.h>
+
+#include "mqtt_state.h"
+
+/****************************************************************/
+
+void IotListDouble_RemoveAllMatches( const IotListDouble_t * const pList,
+				     bool ( *isMatch )( const IotLink_t * const pOperationLink, void * pCompare ),
```

Comment:
I'd follow the PRBR recommendation and declare all function pointers as typedefs at the beginning of the file:
```c
typedef bool ( *MatchFunction_t)( const IotLink_t * const pOperationLink, void * pCompare );

MatchFunction_t isMatch /* Usage. */
```

Reply:
Done.